dali/test/python/test_operator_resize.py

```diff
@@ -670,20 +671,26 @@ def test_checkerboard_dali_vs_onnx_ref():
     out_size = (17, 13)
     out_size_str = '_'.join([str(n) for n in out_size])
     ref_resized_linear_filename = os.path.join(ref_dir, f"checkerboard_linear_{out_size_str}.npy")
-    ref_resized_linear_antialias_filename = os.path.join(ref_dir, f"checkerboard_linear_antialias_{out_size_str}.npy")
+    ref_resized_linear_antialias_filename = os.path.join(
+        ref_dir, f"checkerboard_linear_antialias_{out_size_str}.npy")
     ref_resized_cubic_filename = os.path.join(ref_dir, f"checkerboard_cubic_{out_size_str}.npy")
-    ref_resized_cubic_antialias_filename = os.path.join(ref_dir, f"checkerboard_cubic_antialias_{out_size_str}.npy")
+    ref_resized_cubic_antialias_filename = os.path.join(
+        ref_dir, f"checkerboard_cubic_antialias_{out_size_str}.npy")
 
     # Reference generated with ONNX reference code. To regenerate uncomment
-    # from onnx.backend.test.case.node.resize import interpolate_nd, linear_coeffs, linear_coeffs_antialias, \
-    #                                                 cubic_coeffs, cubic_coeffs_antialias
-    # ref_resized_linear = interpolate_nd(checkerboard, lambda x, _: linear_coeffs(x), output_size=out_size)
+    # from onnx.backend.test.case.node.resize import \
+    # interpolate_nd, linear_coeffs, linear_coeffs_antialias, cubic_coeffs, cubic_coeffs_antialias
+    # ref_resized_linear = interpolate_nd(
+    # checkerboard, lambda x, _: linear_coeffs(x), output_size=out_size)
     # np.save(ref_resized_linear_filename, ref_resized_linear)
-    # ref_resized_linear_antialias = interpolate_nd(checkerboard, linear_coeffs_antialias, output_size=out_size)
+    # ref_resized_linear_antialias = interpolate_nd(
+    # checkerboard, linear_coeffs_antialias, output_size=out_size)
     # np.save(ref_resized_linear_antialias_filename, ref_resized_linear_antialias)
-    # ref_resized_cubic = interpolate_nd(checkerboard, lambda x, _: cubic_coeffs(x, A=-0.5), output_size=out_size)
+    # ref_resized_cubic = interpolate_nd(
+    # checkerboard, lambda x, _: cubic_coeffs(x, A=-0.5), output_size=out_size)
     # np.save(ref_resized_cubic_filename, ref_resized_cubic)
-    # ref_resized_cubic_antialias = interpolate_nd(checkerboard, lambda x, scale: cubic_coeffs_antialias(x, scale, A=-0.5), output_size=out_size)
+    # ref_resized_cubic_antialias = interpolate_nd(
+    # checkerboard, lambda x, scale: cubic_coeffs_antialias(x, scale, A=-0.5), output_size=out_size)
     # np.save(ref_resized_cubic_antialias_filename, ref_resized_cubic_antialias)
```

Comment:
Oh, no! This was _code_ inside - now it's not valid.

Reply:
The code violated the linter, therefore I've broken it into separate lines. I have no way of testing the code, best I can do is to break it better or ask author (@jantonguirao) to give some guidelines. This is rather fresh code, so it shouldn't be a problem

Reply:
The proposal with "prettier" formatting (although I like the one in the PR better)
```python
    # from onnx.backend.test.case.node.resize import interpolate_nd, linear_coeffs, \
    #     linear_coeffs_antialias, cubic_coeffs, cubic_coeffs_antialias
    # 
    # ref_resized_linear = interpolate_nd(checkerboard, lambda x, _: linear_coeffs(x),
    #                                     output_size=out_size)
    # np.save(ref_resized_linear_filename, ref_resized_linear)
    # ref_resized_linear_antialias = interpolate_nd(checkerboard, linear_coeffs_antialias,
    #                                               output_size=out_size)
    # np.save(ref_resized_linear_antialias_filename, ref_resized_linear_antialias)
    # ref_resized_cubic = interpolate_nd(checkerboard, lambda x, _: cubic_coeffs(x, A=-0.5),
    #                                    output_size=out_size)
    # np.save(ref_resized_cubic_filename, ref_resized_cubic)
    # ref_resized_cubic_antialias = interpolate_nd(checkerboard,
    #                                              lambda x, scale: cubic_coeffs_antialias(x, scale,
    #                                                                                      A=-0.5),
    #                                              output_size=out_size)
    # np.save(ref_resized_cubic_antialias_filename, ref_resized_cubic_antialias)
```

Reply:
I prefer your second proposal. It should look like code, just commented out. You can easily uncomment, autoformat and comment back, if you wish

Reply:
Applied the second one